CMakePresets.json

```diff
@@ -249,6 +249,18 @@
         "CMAKE_TOOLCHAIN_FILE": "${sourceDir}/cmake/toolchains/android/x86.cmake"
       }
     },
+    {
+      "name": "emscripten-base",
+      "inherits": ["linux-base"],
+      "hidden": true,
+      "environment": {
+        "CXXFLAGS": "$penv{CXXFLAGS} -msimd128 -msse4.2 -mavx",
+        "LDFLAGS": "$penv{LDFLAGS} -msimd128 -msse4.2 -mavx -Wno-limited-postlink-optimizations -sALLOW_MEMORY_GROWTH=1 -sALLOW_TABLE_GROWTH=1"
```

Comment:
What's the reasoning behind enabling SSE 4.2 and AVX? Godot targets SSE2.

Reply:
Not much reasoning, haha. I'm a bit out of my depth in certain areas of this. Learning as I go! My thought was that targeting the "best" option supported by emscripten would be optimal, since they'd need to be downgraded by the VM if unsupported by the platform anyway. 

Pretty sure I nabbed this from the Jolt js port? 

Anyway, I can change this to mirror Godot.

Reply:
Hmm. It seems if I don't either disable SSE completely or pass -msse4, jolt (the upstream library, not this plugin) fails to compile.

For now, I'll remove the avx flag and keep sse4.